packages/@ionic/cli/src/commands/capacitor/copy.ts

```diff
@@ -14,6 +15,12 @@ export class CopyCommand extends CapacitorCommand implements CommandPreRun {
         type: Boolean,
         default: true,
       },
+      {
+        name: 'inline',
+        summary: 'Use inline source maps (only available on capacitor 4.2.0)',
+        type: Boolean,
+        default: true
```

Comment:
The default should be false right @jcesarmobile @IT-MikeS ?
That way if you omit the option it won't inline source code and if you specify --inline it will work

Reply:
yeah, should be false.
making it true makes the `--help` parameter to show it as `--no-inline` instead of `--inline` and the description makes no sense then.
I've updated it